Sources/ForageSDK/Component/ForagePINTextField/Protocol/VaultWrapper.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  VaultWrapper.swift
+//  
+//
+//  Created by Shardendu Gautam on 6/6/23.
+//
+
+import UIKit
+
+/// The state that the underlying vault exposes during events or statically as input instance attributes.
+internal protocol InternalObservableState {
+    /// isFirstResponder is true if the input is focused, false otherwise.
+    var isFirstResponder: Bool { get }
+    
+    /// isBlured is true if the input is blured, false otherwise.
+    var isBlured: Bool { get }
```

Comment:
not important now but does iOS still refer to it as blur? The use of `isFirstResponder` is new to me so I'm not sure

Reply:
I see no mention of the `blur` concept in Swift, so I removed this.